src/parseTools.js

```diff
@@ -51,12 +51,14 @@ function preprocess(text, filenameHint) {
           var truthy = !!eval(after);
           showStack.push(truthy);
         } else if (line.indexOf('#include') === 0) {
-          var filename = line.substr(line.indexOf(' ')+1);
-          if (filename.indexOf('"') === 0) {
-            filename = filename.substr(1, filename.length - 2);
+          if (showStack.indexOf(false) === -1) {
```

Comment:
Might be good to model this as a helper closure with a name, something like `isTextEnabled`? `notIfdeffedOut`? (naming is hard)
Not being familiar with this code, I had to look around the function to figure out what `showStack` meant (aside: I'm not sure if showStack is a great name for this), and then I had to think about whether `.indexOf(false) === -1` was correct, as opposed to `if (showStack[showStack.length - 1])`, so there's a bit of subtlety here.
My first thought was "could use a comment", but we indexOf(false) two more times, so it might be worth a helper function.

Reply:
Not sure what you mean for a helper function to do here?